doc/new-guidelines.md

```diff
@@ -230,6 +229,49 @@ Properties must be set on view level
     [reanimated/view {:style (style/circle-container opacity)}]))
 ```
 
+### `apply-animations-to-style` vs `[]`
+
+`apply-animations-to-style` is a function wrapping `reanimated/use-animated-style` to make it work in 
```

Comment:
@ulisesmac, the guidelines have grown and the more we increase it, the less devs will read it over and over. I have seen this problem dozens of times already, people missing what's already in the guidelines.

So in that spirit, I'm thinking if we can make this more concise and to the point.

Here's just one suggestion. I removed a little bit of code from the example and changed the text to start with an actionable verb *prefer*.

```
### Pass a vector of styles to Reanimated view

Prefer to pass a vector of styles to `react-native.reanimated/view` `:style`
prop instead of using `apply-animations-to-style` directly. For more details
about Reanimated inline styles, check out Reanimated docs about [inline
styles](https://docs.swmansion.com/react-native-reanimated/docs/fundamentals/your-first-animation/#defining-a-shared-value)
and [useAnimatedStyle](https://docs.swmansion.com/react-native-reanimated/docs/fundamentals/animating-styles-and-props/#animating-styles)
for more information.

(defn f-view []
  (let [scroll-x (reanimated/use-shared-value 0)
        opacity  (reanimated/interpolate scroll-x [0 45 50] [1 1 0])]
    [reanimated/view
     ;; bad
     {:style (reanimated/apply-animations-to-style
              {:opacity   opacity
               :transform [{:translate-x scroll-x}]}
              {:flex-direction :row})}

     ;; good
     {:style [{:opacity   opacity
               :transform [{:translate-x scroll-x}]}
              {:flex-direction :row}]}

     ;; other valid and good variants
     {:style [{:opacity opacity}
              {:transform [{:translate-x scroll-x}]}
              {:flex-direction :row}]}

     {:style {:opacity        opacity
              :transform      [{:translate-x scroll-x}]
              :flex-direction :row}}]))
```

Reply:
I totally agree with this. When I check the guidelines, I usually do a quick search and look for something like 'do not do _this_, do that `instead`.' Thus, the more concise the guideline, the better.

Reply:
Thanks for the suggestion @ilmotta !